dev/large-requirements.txt

```diff
@@ -1,5 +1,5 @@
 ## Large test reqs
 ## Test-only dependencies
-matplotlib==3.2.1
-tf2onnx==1.5.4
-pyarrow==0.17.0
+matplotlib
+tf2onnx
+pyarrow
```

Comment:
`pyarrow` isn't actually a test-only dependency. it's used here: https://github.com/mlflow/mlflow/blob/c9856e7855c9d5340cc0c7b0ea67aba5d29ba075/mlflow/store/artifact/hdfs_artifact_repo.py#L167.

Reply:
Strange, it's not listed in `setup.py`. I've added it there.

Reply:
It should be listed in the `extras` section, rather than `install_requires`, since HDFS artifact repository is not critical to the operation of MLflow in most cases.